README.md

```diff
@@ -97,9 +97,9 @@ Factory method to create a new instance of a responder server
 #### listen()
 start listening for incoming requests
 
-Class: Responder
+Interface: Responder
 ---------------------------------------------------
-Component which holds original request message and create response base on it.
+Implementation of this interface holds original request message and create response base on it.
```

Comment:
Does a client of this class has to know about these hairy details that there's some message inside the responder? Maybe all he needs to know that there's some magic behind send and sendAck that make them work.